$NetBSD: patch-ay,v 1.2 2008/06/22 22:41:01 heinz Exp $

  Change default behaviour of SpamAssassin for pkgsrc in order to shut up
  unwanted warnings in the log files.
  If the users want to use SPF, they can enable the plugin.

--- rules/init.pre.orig	2006-03-10 20:30:02.000000000 +0100
+++ rules/init.pre
@@ -25,5 +25,8 @@ loadplugin Mail::SpamAssassin::Plugin::H
 
 # SPF - perform SPF verification.
 #
-loadplugin Mail::SpamAssassin::Plugin::SPF
+# By default SPF is _not_ a prerequisite for SpamAssassin so we disable it
+# in pkgsrc to avoid confusing warnings in the log file.
+# If you use SPF, just reenable it.
+#loadplugin Mail::SpamAssassin::Plugin::SPF
 
